greengrass_ipc/source/GreengrassCoreIpcModel.cpp

```diff
@@ -5933,6 +7016,253 @@ namespace Aws
+            Aws::Crt::Allocator *allocator) noexcept
+            : ClientOperation(connection, nullptr, operationContext, allocator)
+        {
+        }
+
+        std::future<RpcError> VerifyClientDeviceIdentityOperation::Activate(
+            const VerifyClientDeviceIdentityRequest &request,
+            OnMessageFlushCallback onMessageFlushCallback) noexcept
+        {
+            return ClientOperation::Activate(static_cast<const AbstractShapeBase *>(&request), onMessageFlushCallback);
+        }
+
+        Aws::Crt::String VerifyClientDeviceIdentityOperation::GetModelName() const noexcept
+        {
+            return m_operationModelContext.GetOperationName();
+        }
+
+        AuthorizeClientDeviceActionOperationContext::AuthorizeClientDeviceActionOperationContext(
+            const GreengrassCoreIpcServiceModel &serviceModel) noexcept
+            : OperationModelContext(serviceModel)
+        {
+        }
+
+        Aws::Crt::ScopedResource<AbstractShapeBase> AuthorizeClientDeviceActionOperationContext::
+            AllocateInitialResponseFromPayload(Aws::Crt::StringView stringView, Aws::Crt::Allocator *allocator) const
+            noexcept
+        {
+            return AuthorizeClientDeviceActionResponse::s_allocateFromPayload(stringView, allocator);
+        }
+
+        Aws::Crt::ScopedResource<AbstractShapeBase> AuthorizeClientDeviceActionOperationContext::
+            AllocateStreamingResponseFromPayload(Aws::Crt::StringView stringView, Aws::Crt::Allocator *allocator) const
+            noexcept
+        {
+            (void)stringView;
+            (void)allocator;
+            return nullptr;
+        }
+
+        Aws::Crt::String AuthorizeClientDeviceActionOperationContext::GetRequestModelName() const noexcept
+        {
+            return Aws::Crt::String("aws.greengrass#AuthorizeClientDeviceActionRequest");
+        }
+
+        Aws::Crt::String AuthorizeClientDeviceActionOperationContext::GetInitialResponseModelName() const noexcept
+        {
+            return Aws::Crt::String("aws.greengrass#AuthorizeClientDeviceActionResponse");
+        }
+
+        Aws::Crt::Optional<Aws::Crt::String> AuthorizeClientDeviceActionOperationContext::
+            GetStreamingResponseModelName() const noexcept
+        {
+            return Aws::Crt::Optional<Aws::Crt::String>();
+        }
+
+        Aws::Crt::String AuthorizeClientDeviceActionOperationContext::GetOperationName() const noexcept
+        {
+            return Aws::Crt::String("aws.greengrass#AuthorizeClientDeviceAction");
+        }
+
+        std::future<AuthorizeClientDeviceActionResult> AuthorizeClientDeviceActionOperation::GetResult() noexcept
+        {
+            return std::async(std::launch::deferred, [this]() {
+                return AuthorizeClientDeviceActionResult(GetOperationResult().get());
+            });
+        }
+
+        AuthorizeClientDeviceActionOperation::AuthorizeClientDeviceActionOperation(
```

Comment:
Same general remark on the constructors: A comment would be nice but not required.